rust/scx_loader/Cargo.toml

```diff
@@ -12,7 +12,7 @@ clap = { version = "4.1", features = ["derive", "env", "unicode", "wrap_help"] }
 colored = "2"
 ctrlc = { version = "3.1", features = ["termination"] }
 log = "0.4.17"
-nix = { features = ["process", "signal"], default-features = false, version = "0.29" }
+libc = "0.2.137"
```

Comment:
In the [lock file](https://github.com/sched-ext/scx/blob/main/Cargo.lock#L1370) it seems to be a newer version.